install_local_without_config.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
```

Comment:
I like the idea of a quick install without config! However I think we can remove this as it can be out of scope for this PR.

Some suggestions for this though:
- Maybe we can reuse the `install.py` code to avoid maintaining 2 installation files. You can use a similar approach to the connectors with s3/minio.
- Another approach instead of a different script is just to set the default values to be the local filesystem that way users can do `install.py` and hit enter for all questions without thinking so much to get a full installation with basic configs.
- Even though local file system sounds like a good default, I still think it's important that we check the existence of MinIO on the users machine. Otherwise, new users might think they just need a raw filesystem.

Reply:
OK, commit has been removed.